CHANGELOG.md

```diff
@@ -18,7 +18,8 @@
 
 ### Bug Fixes
 
-* None.
+* Fix false positives in `redundant_discardable_let` when `ignore_swiftui_view_bodies` is enabled.
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown:
```suggestion
* Fix false positives of `redundant_discardable_let` rule in `@ViewBuilder` functions,
  `#Preview` macro bodies and preview provides when `ignore_swiftui_view_bodies` is
  enabled.  
```